composer.json

```diff
@@ -123,7 +126,7 @@
         "mikey179/vfsstream": "^1.6",
         "phake/phake": "@stable",
         "phpunit/phpunit": "~5.7",
```

Comment:
Update this version too, this phpunit version is really outdated

Reply:
this is not required, but I'm not against it: should we do it on this pull request too?

Reply:
Yes, we must update all dependencies (as much as we can cf: the mail I forwarded you)

Reply:
Your mail (if we're talking about the same) don't mention PHPUnit.

Let's discuss it Wednesday with @eternoendless: I have the feeling we're not quite sure of the scope of this PHP update.

Don't get me wrong: I'm also in favor of upgrading PHPUnit to let's say PHPUnit 7 at least (8 requires PHP 7.2), but I'm worried about having to maintain a BIG pull request while my working time is not really dedicated to development tasks 😟.

We can still complete this one with more later 👍

Reply:
phpunit code coverage is not compatible with 7.3 so we need to update it 😎

Reply:
we don't use phpunit code coverage :trollface:

Reply:
I use it on facetedsearch, and it's already a dependency of phpunit https://packagist.org/packages/phpunit/phpunit#5.7.27 :trollface:

Reply:
I fear updating phpunit could break existing tests, can we do that on another PR?

Reply:
I'm also in favor of doing the upgrade of PHPUnit tests in another PR because if "I" do it, I'll use the upgrade of PHPUnit version to review some use of assertions deprecated in the latest versions.

So the upgrade may create a big pull request